iris/src/iris/retrieval/faq_retrieval.py

```
@@ -27,9 +27,12 @@ class FaqRetrieval(BaseRetrieval):
     retrieval pipelines (basic and hypothetical), and merges them to return the final FAQ list.
     """
 
-    def __init__(self, client: WeaviateClient, **kwargs):
+    def __init__(self, client: WeaviateClient, local: bool = False, **kwargs):
```

Ruff (0.14.11)

Unused method argument: `kwargs`

(ARG002)